coreth.go

```diff
@@ -19,6 +19,7 @@ import (
 	"github.com/ethereum/go-ethereum/ethdb"
 	"github.com/ethereum/go-ethereum/event"
 	"github.com/ethereum/go-ethereum/trie"
+
 	//"github.com/ethereum/go-ethereum/event"
```

Comment:
Why do we have commented out imports?

Reply:
No idea. Going to remove the commented out imports.